argilla-frontend/components/features/annotation/container/questions/QuestionsForm.vue

```diff
@@ -179,7 +184,11 @@ export default {
         return "--focused-form";
     },
     formHasFocus() {
-      return this.autofocusPosition || this.autofocusPosition == 0;
+      if (!this.$platform.isMobile) {
```

Comment:
The nested if condition can be simplified to improve readability. Consider using an early return pattern.